.github/workflows/merged.yml

```diff
@@ -258,7 +258,68 @@ jobs:
             certificate_password:$CERTIFICATE_PASSWORD \
             apikey_path:$APIKEY_PATH --verbose
 
+  deploy_build_app:
```

Comment:
is this a duplicate of the other step? should we create a separate `yaml` (with parameters, if required) and reuse, rather than duplicating?

Reply:
It largely is yes apart from the
```yml
- name: Export AppCheck Secret
  run: |
    touch ../secrets.xcconfig
```
in this "build" job.
Perhaps we can do that conditionally based on a parameter then..?

Reply:
Yeah, or extract the bits that aren't that into another file and call it from both 🙂